llvm/test/lab2/ivanov_nikita/test.ll

```diff
@@ -0,0 +1,44 @@
+; RUN: opt -passes=ivanov-loop-pass -S %s 2>&1 | FileCheck %s
+
+; void foo(int n, int m) {
+;     int c0;
+;     int c1;
+;     for (c0 = n; c0 > 0; c0--) {
```

Comment:
Could you please add the following test cases:
- while, do-while loops
- no loops but if-else condition present (check that no insertion is done)
- function with any loop and already added `loop_*` calls